circle.yml

```diff
@@ -5,7 +5,7 @@ general:
 machine:
   # Add some environment variables
   environment:
-   HOST: $(case $CIRCLE_NODE_INDEX in 0) host='' ;; 1) host='i686-w64-mingw32-' ;; esac; echo $host)
+    CROSS_COMPILE: $([[ $CIRCLE_NODE_INDEX -eq 1 ]] && echo 'i686-w64-mingw32-')
 
```

Comment:
Sounds good to me, I just thought it would be a little clearer without the `case` statement and the intermediate variable; for more than 2 kinds of host it's definitely better as a `case` statement.